MediaContentSynchronizationCms/composer.json

```diff
@@ -8,6 +8,9 @@
         "magento/module-media-gallery-synchronization-api": "*",
         "magento/module-media-content-api": "*"
     },
+    "suggest": {
+         "magento/module-media-content-synchronization": "*"
```

Comment:
I think it's worth to introduce the `Magento\MediaContentSynchronizationApi\Model\Entities` class to be able to use the `magento/module-media-gallery-synchronization-api` for configuration. (I.e. the same way we have `\Magento\MediaContentApi\Model\Composite\GetEntityContents` as part of API module)

In this case:
 - `Magento\MediaContentSynchronizationApi\Model\GetEntitiesInterface` should be introduced as an SPI service
 - `Magento\MediaContentSynchronizationApi\Model\Entities` should be introduced for DI configuration (accepting the configuration to the constructor and implementing the `GetEntitiesInterface`)
 - `GetEntitiesInterface` should be used in the `RemoveObsoleteContentAsset`